client/src/main/java/org/apache/rocketmq/client/impl/MQClientAPIImpl.java

```diff
@@ -577,12 +577,12 @@ public void operationComplete(ResponseFuture responseFuture) {
                     }
                 } else {
                     if (!responseFuture.isSendRequestOK()) {
-                        pullCallback.onException(new MQClientException("send request failed", responseFuture.getCause()));
```

Comment:
But this is not a sl4j 's logging, it's a exception, so direct string is needed.